### PR TITLE
refactor: HttpsHelloWorld

### PR DESCRIPTION
**Pull request by dinojohn**
Changed the input type of SSLContexBuilder and swapped the old keystore and certifiacte with a new cert and key.

### CODE REVIEW

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 433520.29**

File: example/src/main/scala/example/HttpsHelloWorld.scala [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
-import zhttp.service.server.ServerSSLHandler.{ServerSSLOptions, ctxFromKeystore}
+import zhttp.service.server.ServerSSLHandler._
 import zhttp.service.{EventLoopGroup, Server}
 import zio.{App, ExitCode, URIO}
 
 object HttpsHelloWorld extends App {
-
   // Create HTTP route
-  val app: HttpApp[Any, Nothing] = Http.collect[Request] {
+  val app: HttpApp[Any, Nothing] = Http.collect {
```
Comment (amitksingh1490) [inline example/src/main/scala/example/HttpsHelloWorld.scala]: Unnecessary  change

Comment (dinojohn) [reply example/src/main/scala/example/HttpsHelloWorld.scala]: reverting

File: zio-http/src/main/scala/zhttp/service/server/ServerSSLHandler.scala [hunk 2]
```diff
@@ -56,7 +56,7 @@ object ServerSSLHandler {
       .build()
   }
 
-  def ctxFromCert(certFile: File, keyFile: File): SslContext = {
+  def ctxFromCert(certFile: InputStream, keyFile: InputStream): SslContext = {
```
Comment (amitksingh1490) [inline zio-http/src/main/scala/zhttp/service/server/ServerSSLHandler.scala]: This needs to be either file or path

**Review by amitksingh1490 — CHANGES_REQUESTED** [review]

File: example/src/main/scala/example/HttpsHelloWorld.scala [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
-  val sslctx = ctxFromKeystore(getClass.getResourceAsStream("mysslstore.jks"), "password", "password")
+  val sslctx = ctxFromCert(
+    getClass().getClassLoader().getResourceAsStream("server.crt"),
+    getClass().getClassLoader().getResourceAsStream("server.key"),
+  )
 
   private val server =
-    Server.port(8090) ++ Server.app(app) ++ Server.ssl(ServerSSLOptions(sslctx))
+    Server.port(8090) ++ Server.app(app) ++ Server.ssl(
+      ServerSSLOptions(sslctx, SSLHttpBehaviour.Accept),
```
Comment (amitksingh1490) [inline example/src/main/scala/example/HttpsHelloWorld.scala:29]: Why is this needed?

File: example/src/main/scala/example/HttpsHelloWorld.scala [hunk 4]
```diff
@@ -1,29 +1,33 @@
...
   /**
    * sslcontext can be created using SslContexBuilder. In this example an inbuilt API using keystore is used. For
-   * testing this example using curl, setup the certificate named "localhost.cert" from resources for the OS.
-   * Alternatively you can create the keystore and certificate using the following link
+   * testing this example using curl, setup the certificate named "server.crt" from resources for the OS. Alternatively
+   * you can create the keystore and certificate using the following link
    * https://medium.com/@maanadev/netty-with-https-tls-9bf699e07f01
    */
-  val sslctx = ctxFromKeystore(getClass.getResourceAsStream("mysslstore.jks"), "password", "password")
+  val sslctx = ctxFromCert(
```
Comment (amitksingh1490) [inline example/src/main/scala/example/HttpsHelloWorld.scala:22]: Why is this required

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 433097.34**

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 446295.20**

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 424627.49**

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 432752.61**

**Review by amitksingh1490 — APPROVED** [review]

**Comment by amitksingh1490** [comment]
## 🚀🚀🚀 Benchmark Results 
 **Concurrency: 256** 
 **Requests/sec: 434524.96**
